src/tools/ScenarioMeasurement/Startup/Startup.cs

```diff
@@ -120,7 +120,14 @@ static void checkArg(string arg, string name)
             if (!failed)
             {
                 logger.Log("Parsing..");
-                TraceEventSession.Merge(new[] { kernelTraceFile, userTraceFile }, traceFileName);
+                if (File.Exists(userTraceFile))
+                {
+                    TraceEventSession.Merge(new[] { kernelTraceFile, userTraceFile }, traceFileName);
+                }
+                else
+                {
+                    TraceEventSession.Merge(new[] { kernelTraceFile }, traceFileName);
+                }
```

Comment:
What about something like:
```cs
var files = new List<string>();
files.Add(kernelTraceFile);
if(File.Exists(userTraceFile))
{
    files.Add(userTraceFile);
}
TraceEventSession.Merge(files.ToArray(), traceFileName);
```

Reply:
Yah this looks nicer

Reply:
second thought: this could be a learning to me and I wonder more specifically why importing and creating a list object  is better than using the if-else in terms of efficiency. Reasons I can think of: 1)  it's easier to add more than two files in the future.  2) avoid writing repetitive code.

Reply:
Those are exactly the reasons I suggested this change. This approach is less efficient in terms of performance, but this isn't an area where that matters and the difference is trivial in this case.